docs/REST API/rest-api.md

```diff
@@ -8786,7 +8786,7 @@ Get a list of all software.
       },
       "app_store_app": null,
       "versions_count": 3,
-      "source": "apps",
+      "source": "apps", # ios,ipados
```

Comment:
@noahtalerman the below fields are `ios_apps` and `ipados_apps`, are these intentionally different?